### ISSUE TITLE
Tier checkboxes bugs

### ISSUE THREAD

**Issue by sciepsilon** [Designist/Korpus#100]
Sometimes tiers show up doubled, or tiers that should be hidden are shown, usually for only some of the sentences. This behavior seems non-deterministic. Clicking around on the checkboxes for tiers to show always fixes it eventually. This behavior has been observed on multiple stories; in 002 parts of speech showed up for some sentences upon loading; in 003 the words tier and "citation form" tier (or other duplicate morpheme tier) showed up when their checkboxes weren't checked, although I think not at the same time.

**Comment by sciepsilon** [comment]
![image](https://user-images.githubusercontent.com/19783656/28379287-2908636c-6c81-11e7-9db4-6e34b38db299.png)
This showed up immediately upon loading the story. (I got there by changing the url from another story I was viewing. The previous story did not have parts of speech shown.) Clicking around on the checkboxes did nothing to the aberrant part-of-speech tier *until* I clicked the part-of-speech checkbox (correctly showing everything I think) and unchecked it (correctly hiding everything).
